bin/cli.js

```diff
@@ -31,9 +34,11 @@ var model = runner.init(argv);
 
 runner.run(model, function(warnings) {
   warnings.map(runner.warnModule(model));
-  var status = runner.runElmTest(model);
-  if (status === 0) runner.cleanup(model);
-  runner.warnSummary(model, warnings);
-  process.exit(status);
+  if (model.run) {
+    var status = runner.runElmTest(model);
+    if (status === 0) runner.cleanup(model);
+    runner.warnSummary(model, warnings);
```

Comment:
I'm not sure if I remember correctly, but I think we want this in either case, right?

Reply:
Yeah, I think so.